### ISSUE TITLE
Incorrect warning for using colors 'black' and 'lightgray'

### ISSUE THREAD

**Issue by adnanhemani** [python-visualization/folium#1170]
#### Please add a code sample or a nbviewer link, copy-pastable if possible

```python
#an example, but throws for anywhere where the color 'black' or 'lightgray' is passed in as an argument
icon_args = {"color": "black"}
f = folium.Icon(**icon_args)
```
#### Problem description

It seems that the colors "lightgray" and "black" are merged together in the `color_options` list (as "lightgrayblack") but in the argument documentation are correctly listed as "light gray" and "black". This seems to be throwing an (incorrect) warning to the user:

<img width="745" alt="Screen Shot 2019-06-24 at 7 52 18 AM" src="https://user-images.githubusercontent.com/10746770/59988585-00201580-9655-11e9-9243-305bd8aed19a.png">

From the documentation:
<img width="903" alt="Screen Shot 2019-06-24 at 7 52 44 AM" src="https://user-images.githubusercontent.com/10746770/59988650-4bd2bf00-9655-11e9-8b46-ab9e91f80116.png">

#### Expected Output
No user warning for using "black" or "lightgray".

#### Output of ``folium.__version__``
0.9.1

**Comment by Conengmo** [comment]
That's a typo indeed :( Thanks for the report! Do you want to make a PR with a fix? Ideally, we should update [this test](https://github.com/python-visualization/folium/blob/f05f3ce9b3d71f87a0738f3c8c1236b933e497a9/tests/test_map.py#L94) to make sure this issue fails the test.

**Comment by adnanhemani** [comment]
> That's a typo indeed :( Thanks for the report! Do you want to make a PR with a fix? Ideally, we should update [this test](https://github.com/python-visualization/folium/blob/f05f3ce9b3d71f87a0738f3c8c1236b933e497a9/tests/test_map.py#L94) to make sure this issue fails the test.
> 
> By the way, I read the issue here: [data-8/datascience#385](https://github.com/data-8/datascience/pull/385)
> With this issue and [this one](https://github.com/python-visualization/folium/issues/1168) closed we can release 0.9.2. It will contain bug fixes, and a version bump of Leaflet with no breaking changes. I hope that is acceptable for your project. Here's a list of commits since 0.9.1: [v0.9.1...master](https://github.com/python-visualization/folium/compare/v0.9.1...master)

Yes, I think that should be fine. I'll push out a PR as soon as I can - getting the dev environment setup is taking me a while. Thanks for the fast action!

**Comment by adnanhemani** [comment]
I've made the PR. Not sure how else better to update the tests, to be honest. Thanks for the timely response!

**Comment by Conengmo** [comment]
#1171 has been merged.

**Comment by Conengmo** [comment]
@adnanhemani we just released a new version of folium, 0.10.0. It also includes your fix. It's not a patch release because it includes a new plugin and we bumped Leaflet from 1.4.0 to 1.5.1. Other than that it's only fixes, so I hope you can upgrade without any issues.

**Comment by adnanhemani** [comment]
Sure, I'll try it out when I get some time. Thanks!
